src/components/Editor.singleton.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Get instance of Editor component
```

Comment:
Might be nice to have a comment why we need this for later reference with a link to the issue or y.js code